extensions/wikia/CommunityPage/styles/components/_insights-module.scss

```diff
@@ -1,77 +1,87 @@
+$module-width: 239px;
 
-$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen.svg';
+.community-page-insights-module {
+	@include flexbox();
+	@include flex-direction(column);
 
-@if $is-dark-wiki {
-	$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen-dark.svg';
-}
+	background-color: $color-page;
+	border: 1px solid $color-page-border-subtle;
+	margin-bottom: $content-spacing-regular;
+	margin-right: $content-spacing-small;
+	width: $module-width;
 
-.community-page-insights-module {
-	margin-bottom: 34px;
+	.community-page-insights-module-article-data {
+		color: $color-links;
+	}
 
-	.community-page-insights-module-header {
-		font-size: 24px;
-		font-weight: bold;
-		line-height: 28px;
-		margin: 0 0 12px;
+	.community-page-insights-module-footer {
+		@include flexbox();
 	}
 
-	.community-page-insights-module-list {
-		list-style: none;
-		margin: 0;
+	.community-page-insights-module-full-list-link {
+		@include flex-grow(1);
+
+		color: $color-buttons;
+		display: block;
+		font-size: $typescale-size-minus-2;
+		font-weight: bold;
+		line-height: $typescale-size-plus-6;
+		margin-right: $content-spacing-large;
+		text-align: right;
+		text-transform: uppercase;
 	}
 
-	.community-page-insights-module-list-item {
+	.community-page-insights-module-header {
+		@include align-items(center);
 		@include flexbox();
 
-		align-items: center;
-		border-bottom: 1px solid $color-page-border-subtle;
-		padding: 20px;
+		background: $color-buttons;
+		color: $color-button-text;
+		font-size: $typescale-size-minus-1;
+		font-weight: bold;
+		line-height: $typescale-size-base;
+		margin: 0 0 $content-spacing-large;
+		padding: $content-spacing-regular;
+		text-transform: uppercase;
 	}
 
-	.community-page-insights-module-edit-icon {
-		background: url($edit-icon) no-repeat;
-		height: $item-icon-size;
-		margin-right: $item-icon-margin-right;
-		width: 24px;
+	.community-page-insights-module-header-icon {
+		fill: $color-buttons;
+		margin-right: $content-spacing-small;
 	}
 
-	.community-page-insights-module-article-data {
-		color: $color-links;
-		overflow: hidden;
-		text-overflow: ellipsis;
-		white-space: nowrap;
-		width: $item-data-width;
+	.community-page-insights-module-help {
+		height: $element-size;
+		width: $element-size;
 	}
 
-	.community-page-insights-module-metadata {
-		color: $color-alternate-text;
-		font-size: 12px;
-
-		.communitypage-user-link {
-			display: inline-block;
-			max-width: 140px;
-			overflow: hidden;
-			text-overflow: ellipsis;
-			vertical-align: bottom;
-		}
+	.community-page-insights-module-help-icon {
+		fill: $color-page-border-subtle;
+		margin: $content-spacing-small-plus $content-spacing-small;
 	}
 
-	.community-page-insights-module-edit-link {
-		text-align: right;
-		width: $item-link-width;
+	.community-page-insights-module-list {
+		@include flex-grow(1);
+
+		font-size: $typescale-size-base;
+		list-style: none;
+		margin: 0;
+		padding: 0 $content-spacing-regular;
```

Comment:
for those above you can use `reset` class: 
`ul.reset, ul.reset li {
	list-style: none;
	margin: 0;
	padding: 0;`